code/game/objects/items/devices/scanners.dm

```diff
@@ -359,8 +359,6 @@ GENE SCANNER
 			mutant = TRUE
 		else if (S.mutantstomach != initial(S.mutantstomach))
 			mutant = TRUE
-		else if (S.flying_species != initial(S.flying_species))

```

Comment:
Should you not replace this with mutant_wings check to see if they are a mutant?

Reply:
that is unecessary. This checks if the user has their species organ var changed, which doesnt ever happen with wings. it does not check their actual organs.


besides, scanner gate is never, ever used, ever.